src/data/strategies/eth-btc-trend.ts

```diff
@@ -25,6 +25,10 @@ export const ethBtcTrend = {
     protocol: 2.5,
     "strategy provider": 7.5,
   },
+  staking: {
+    endDate: new Date("2022-12-12"),
```

Comment:
this is local time date, We should use `zonedTimeToUtc(new Date("2022-12-12"), "UTC")` from `date-fns-tz`